src/data/reducers/utils.test.js

```diff
@@ -22,16 +22,28 @@ describe('Reducer Utils', () => {
       expectedAssets[1].loadingFields = [assetLoading.LOCK];
       expect(addLoadingField(assets, 'chinchilla', assetLoading.LOCK)).toEqual(expectedAssets);
     });
+    it('adds loading field to existing loading fields', () => {
+      const mockLoadingField = {
+        MOCK: 'MOCK',
+      };
+
+      const newAssets = deepCopyAssets();
+      newAssets[0].loadingFields = [assetLoading.LOCK];
+
+      const expectedAssets = deepCopyAssets();
+      expectedAssets[0].loadingFields = [assetLoading.LOCK, mockLoadingField];
+
+      expect(addLoadingField(newAssets, 'lynx', mockLoadingField)).toEqual(expectedAssets);
+    });
   });
 
-  describe('addLoadingField', () => {
+  describe('removeLoadingField', () => {
```

Comment:
Just fixing a typo, since the tests in this describe block are for the removeLoadingField utility function.